src/behaviours/__tests__/form-test.js

```diff
@@ -0,0 +1,95 @@
+import React, {Component} from 'react';
+import {connect} from '../form';
+import {Provider as StoreProvider} from 'react-redux';
+import store from '../../example/store';
+const {renderIntoDocument} = TestUtils;
+import find from 'lodash/find';
+
+describe('The form connect', () => {
+    let capturedProps;
```

Comment:
On all the error message I would have add a `FormConnect: ` or something like that just to see who is the error sender.
As in the action builder.